### PR TITLE
v2ray-plugin 1.0 (new formula).

### PR DESCRIPTION
**Pull request by icpz**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/v2ray-plugin.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  sha256 "5320eff71d99edcfa1e5e883debb42c50ccdebc2b43eb30c6c1b16baa632cda2"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", bin/"v2ray-plugin"
+  end
+
+  test do
+    output = shell_output("#{bin}/v2ray-plugin -help 2>&1", 2)
```
Comment (javian) [inline Formula/v2ray-plugin.rb]: Would it be possible to write a test that uses some functionality of the software rather than just printing text ?

Comment (lcdtyph) [reply Formula/v2ray-plugin.rb]: Hi, I've updated the commit.

Comment (lcdtyph) [reply Formula/v2ray-plugin.rb]: Oh, sorry for the misunderstand. It's a little complicated to test the functionality, but I'm working on it.

**Comment by javian** [comment]
@BrewTestBot test this please

**Comment by lcdtyph** [comment]
Hello, anything else can be improved?

**Comment by SMillerDev** [comment]
@BrewTestBot test this please

**Comment by SMillerDev** [comment]
> Hello, anything else can be improved?

The test seemed not to have picked up the commits you made. Could you squash them?

**Comment by lcdtyph** [comment]
> > Hello, anything else can be improved?
> 
> The test seemed not to have picked up the commits you made. Could you squash them?

Squashed

**Comment by lcdtyph** [comment]
Hi, all. Is this ready for brew? Can't wait for that 😄

**Comment by lcdtyph** [comment]
@fxcoudert 
Sorry but may I know what's blocking this formula~

**Comment by fxcoudert** [comment]
Looks good to me. @javian ?

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Comment by fxcoudert** [comment]
Thanks @lcdtyph for the pull request!

**Pull request closed without merging**